### PR TITLE
Content/#4 complete form html

### LINKED ISSUE #4: Complete Form HTML
Add the Html according to the decided css framework, to have fancy and nice forms

Also, would be nice to have them appear as you enter them in the form, with preety animations

### PR DESCRIPTION
**Pull request by torch2424**
closes #4 

Video:

untitled recording 3

This creates the basic html and functionality of the form, in how it will act. It still needs java script implementation with pinging the backend, some ng-animate animations, and some responsive features (Though it is already 95% of the way there for responsiveness)

Also, thanks to some tips by @julianpoy , autocompletion also works, I just wanted to showcase filling out the form by hand

### CODE REVIEW

**Comment by davidnuon** [comment]
Is this is a sign-in or sign-up?

**Comment by torch2424** [comment]
@davidnuon A little of both, @rogerhoward Asked for something that can log everyone that came to the event, but also on board people to our orgs, if they are not yet already onboarded :)

**Comment by davidnuon** [comment]
Where is the data being stored?

**Comment by karimamer** [comment]
i Like what i see thanks @torch2424 for your work

**Comment by torch2424** [comment]
@karimamer Your welcome!

@davidnuon the data will be stored in like a google sheets or something that we can all have access to

File: app/views/main.html [hunk 1]
```diff
@@ -1,11 +1,161 @@
...
+
+        <h3 class = "formTitle"
+        ng-if = "formData.slack ||
+        formData.github">
+            Thank you for joining our organizations! You should be receiving an email soon with your invitation!
+        </h3>
+
+        <h3
+        class = "formTitle">
+            Welcome to Code && Coffee Long Beach, and we hope you enjoy your time!
```
Comment (davidnuon) [inline app/views/main.html]: Change this to HTML Entities `&amp;&amp;`

Comment (torch2424) [inline app/views/main.html]: Okay i shall once I get the chance :)

**Comment by torch2424** [comment]
@davidnuon I made your changes :)

**Comment by davidnuon** [comment]
I had more to say, but I realized the scope of this issue is to do just the form HTML. I'll try to see how difficult the github onboarding issue is #6 since that's an obvious addition. 

Looks good, merging.

**Comment by torch2424** [comment]
Sounds good! I am also going to delete the branch since it is no longer needed

**Pull request merged**